Add new libs, clients and deployments links (#4)
nlewo/awesome-gandi: README.md

- [Libraries](#libraries)
- [Deployment](#deployment)
- [Clients](#clients)
- [APIs](#apis)

## Libraries

- [go-gandi](https://github.com/go-gandi/go-gandi) - A Go library for Gandi V5 APIs.
- [Lego](https://github.com/go-acme/lego) - A ACME client and library that supports Gandi V5 API and built in Go.
- [LibDNS](https://github.com/libdns/libdns) - A Universal DNS provider APIs for Go with a [Gandi V5 provider](https://github.com/libdns/gandi) available.

## Deployment

- [Gandi Terraform Provider](https://github.com/go-gandi/terraform-provider-gandi) - A Terraform provider to manages Gandi resources (domains, mailboxes, etc.).

> - [Ansible Gandi LiveDNS](https://docs.ansible.com/ansible/latest/collections/community/general/gandi_livedns_module.html) - A Ansible module for managing Gandi LiveDNS records

Reviewer: ```suggestion
- [Ansible Gandi LiveDNS](https://docs.ansible.com/ansible/latest/collections/community/general/gandi_livedns_module.html) - A Ansible module for managing Gandi LiveDNS records.
```

Reviewer: Hehe, you were faster than me!

> - [Ansible Gandi LiveDNS](https://docs.ansible.com/ansible/latest/collections/community/general/gandi_livedns_module.html) ([source code](https://github.com/ansible-collections/community.general/blob/main/plugins/modules/net_tools/gandi_livedns.py)) - A Ansible module for managing Gandi LiveDNS records

Reviewer: ```suggestion
- [Ansible Gandi LiveDNS](https://docs.ansible.com/ansible/latest/collections/community/general/gandi_livedns_module.html) - A Ansible module for managing Gandi LiveDNS records
```

## Clients

> - [Lego](https://github.com/go-acme/lego) - A ACME client and library that supports Gandi V5 API and built in Go.

Reviewer: ```suggestion
- [Lego](https://go-acme.github.io/lego/usage/cli/) - A ACME client and library that supports Gandi V5 API and built in Go.
```
